api/tools/src/main/java/org/pmiops/workbench/tools/DeleteWorkspaces.java

```diff
@@ -1,246 +1,188 @@
 package org.pmiops.workbench.tools;
 
-import java.io.BufferedReader;
```

Comment:
this is better understood as a deletion of the previous tool DeleteWorkspaces and a rename of the new too DeleteOrphanedWorkspaces